webdriver-spec.html

```diff
@@ -1844,19 +2069,34 @@
 <p>The <dfn>New Session</dfn> <a>command</a>
  creates a new WebDriver <a>session</a> with the <a>endpoint node</a>.
  If the <a>maximum active sessions</a> has been reached,
- that is if the <a>endpoint node</a> already has a <a>current session</a>,
  there is a problem processing the given <a>capabilities</a>,
  or the provisioning of a <a>remote end</a> is impossible,
  a <a>session not created</a> <a>error</a> is returned.
 
 <p>If the <a>remote end</a> is an <a>intermediary node</a>,
- they are allowed to freely use the <a>capabilities</a> data
- e.g. to select a specific browser to test based on
- a combination of the <a data-lt="required-capabilities">required</a>
- and <a>desired capabilities</a>.
- Typically the <a>New Session</a> response from the <a>endpoint node</a>
- selected in this process will then be relayed directly to the <a>local end</a>,
- as outlined in step 1 of the <a>remote end steps</a>.
+ it may use the result of the <a>processing capabilities</a> algorithm to route
+ the new session request to the appropriate <a>endpoint node</a>.
+ An <a>intermediary node</a> may also define custom <a>capabilities</a> to
+ assist in this process, however, these capabilities
+ <em class="rfc2119" title="MUST NOT">MUST NOT</em> be forwarded to the
+ <a>endpoint node</a>. Furthermore, if the <a>intermediary node</a> requires
+ additional information unrelated to user agent features,
+ it is <em class="rfc2119" title="RECOMMENDED">RECOMMENDED</em> that this
+ information be passed as top-level parameters and not part of the requested
+ <a>capabilities</a>.
```

Comment:
Worth noting that endpoint nodes MUST ignore top-level parameters that they don't process.